lib/api.js

```diff
@@ -47,7 +48,7 @@ exports.default = function (_ref) {
   if (!module.exports) {
     // babel did not transform modules
     // eslint-disable-next-line no-underscore-dangle
-    component = module.__esModule ? module.default : module;
+    component = ctx.__esModule ? ctx.default : ctx;
```

Comment:
Yeah, you're right, that's what they (vue-loader) seem to [do](https://github.com/vuejs/vue-loader/blob/master/lib/loader.js#L480). But is it `*.vue` files that don't support named export or `vue-hot-reload-api` that doesn't like it or something ?